app/views/content/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -58,66 +50,100 @@ keywords:
+* Training lasts one 1 academic year full-time, or longer part-time
+* You can apply for paid teaching jobs as soon as you successfully complete your training
+
+## Apply for teacher training, step by step
+
+We can help you make a successful application. [Get support and advice](#get-support-and-advice) explains how.
 
 ## 1. Check your qualifications
 
 To teach children aged 11 to 18 in secondary school, you’ll need:
 
 * a bachelor’s degree (this can be from a university outside the UK)
-* a standard in English and maths equivalent to a grade 4 GCSE 
+* to meet a standard in English and maths equivalent to a grade 4 GCSE
 
-To teach children aged 3 to 11 in primary school, you will also need a standard in physics, chemistry or biology equivalent to a grade 4 GCSE.
+To teach children aged 3 to 11 in primary school, you will also need to meet a standard in physics, chemistry or biology equivalent to a grade 4 GCSE.
 
-A GCSE is an academic qualification awarded for exams in England, usually taken at age 16. [Get help understanding English qualifications and their international equivalents](#get-help-and-advice).
+A GCSE is an academic qualification awarded for exams in England, usually taken at age 16. [We can help you](#get-support-and-advice) show your English, maths and science qualifications are the same standard as a grade 4 GCSE.
+
+If you do not have qualifications that are equivalent to GCSEs, you may be able to sit a test set by training providers to show your knowledge of English, maths and science. [You can contact us for advice about this](#get-support-and-advice).
 
 ### If you do not have a university degree
 
-If you do not have a degree, you can [apply to study a 3 to 4 year undergraduate course in England](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) which combines a teaching degree with a professional teaching qualification ([qualified teacher status, or QTS](https://www.gov.uk/guidance/qualified-teacher-status-qts)). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
+If you do not have a degree, you can [apply to study a 3 to 4 year undergraduate course in England](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) which combines a teaching degree with [qualified teacher status](https://www.gov.uk/guidance/qualified-teacher-status-qts) (QTS). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
 
 Visit the [UK Council for International Student Affairs](https://www.ukcisa.org.uk/) for lots of helpful information about studying at an English university.
 
-## 2. Get help and advice
+## 2. Get support and advice
+
+To get advice about training to teach, you can:
 
-### Help with international qualifications
+* call 0800 389 2500 for free from anywhere in the world, Monday to Friday between 8.30am and 5.30pm, Greenwich Mean Time
```

Comment:
Is this definitely always GMT or should it be UK local time? (i.e., does the helpline observe daylight savings?)

Reply:
I agree. We've changed this.